### PR TITLE
Answered a general question and scanned all docs for spelling/grammar

### PR DESCRIPTION
**Pull request by MillyCodes**
I answered a general question and scanned the rest of the documents, fixing any minor spelling or grammar mistakes.

### CODE REVIEW

**Comment by emmabostian** [comment]
Looks good but please resolve conflicts!

**Comment by MillyCodes** [comment]
Sorry, was working with an older copy from what was just update.

File: general.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+
+An IDE is an essential part of any developers tool kit if you work in front end usually a great code editor is one that features a well-designed, super efficient, and ultra speedy user interface. There are several that do this well, there are also folks who think it's super important but it doesn't really matter. Ultimately it is a tool and none is necessarily better than another or "Best" you just choose the one that will make you a more efficient coder. The one you use is a matter of preference however here is a short list of the most popular.
+
+-   [Visual Studio Code](https://notepad-plus-plus.org/download/v6.9.2.html)
+-   [Notepad++](https://notepad-plus-plus.org/download/v6.9.2.html)
+-   [Sublime Text](https://notepad-plus-plus.org/download/v6.9.2.html)
+-   [Atom](https://atom.io/)
+-   [Brackets](http://brackets.io/)
+
+I've been programming for a bit now and the minimum for front end code editors is that they can assist you with a few essential functions such as syntax highlighting, suggestions to an extent and indentation, some may also have an integrated terminal.
```
Comment (emmabostian) [inline general.md]: Looks great! Could you please just re-phrase in the 3rd person? :) So no "I"

**Review by emmabostian — APPROVED** [review]
Looks great, thanks!

**Pull request merged**